### PR TITLE
Nikto Cheatsheet

### PR DESCRIPTION
**Pull request by pseudozidu**
What does your Instant Answer do?
It will produce a cheatsheet for perl based vulnerability scanner that scans web-server for thousands of vulnerabilities and other known issues. It is advanced and more accurate and people would like the concept.
What is the data source for your Instant Answer? (Provide a link if possible)
r00tsec
http://www.r00tsec.com/2014/06/cheatsheet-nikto-cheat-sheet.html
Offical site of nikto: cirt
https://cirt.net/nikto2-docs/options.html#id2741238
Why did you choose this data source?
Both source are ingredient for good cheatsheet recipe, include some basic and advanced options with installation on different platform.
Are there any other alternative (better) data sources?
Cheatography provide same stuff as r00tsec do
Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)
Sys admin and Security
Is this Instant Answer connected to a DuckDuckHack Instant Answer idea?
Yes, It is cheatsheet
Are you having any problems? Do you need our help with anything?
No, till now i am not having any issue in creating cheatsheet.
Where did you hear about DuckDuckHack? (For first time contributors)
2nd contribution
What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)

image

-----
IA Page: https://duck.co/ia/view/nikto_cheat_sheet

### CODE REVIEW

File: share/goodie/cheat_sheets/json/nikto.json [hunk 1]
```diff
@@ -0,0 +1,358 @@
+{
+    "id": "nikto_cheat_sheet",
+    "name": "Nikto",
+    "description": "A Nikto is perl based vulnerability scanner that scan webservers for thousand of vulnerabilities and other known issues",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/nikto.json]: sp: "Nikto is a Perl based ..."

File: share/goodie/cheat_sheets/json/nikto.json [hunk 2]
```diff
@@ -0,0 +1,358 @@
...
+            {
+                "val": "Install Nikto on Ubuntu",
+                "key": "sudo apt-get install nikto"
+            },
+            {
+                "val": "Install Nikto on windows (On windows first install the perl interpreter)",
+                "key": "C:\\pentest\\nikto-2.1.5>perl nikto.pl -h example.com"
+            },
+            {
+                "val": "Baisc Usage",
```
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/nikto.json]: Typo: Basic

**Comment by pseudozidu** [comment]
@moollaza @mintsoft Kindly update on Nikto cheatsheet

**Comment by moollaza** [comment]
@pseudozidu this LGTM :+1:

Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

**Comment by pseudozidu** [comment]
@moollaza Kindly update developer name:- rohitzidu
In beta duckduckgo, I am unable 2 see

**Comment by moollaza** [comment]
@pseudozidu congrats! It's live: https://duckduckgo.com/?q=nikto%20cheat%20sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**